apps/storybook/src/Tile.stories.tsx

```diff
@@ -285,3 +286,118 @@ Folder.args = {
   badge: undefined,
   thumbnail: <SvgFolder />,
 };
+
+export const Status: Story<TileProps> = (props) => {
+  const {
+    name,
+    description,
+    metadata,
+    thumbnail,
+    moreOptions,
+    status,
+    ...rest
+  } = props;
+  return (
+    <Tile
+      name={name}
+      description={description}
+      metadata={metadata}
+      thumbnail={thumbnail}
+      moreOptions={moreOptions}
+      status={status}
+      {...rest}
+    />
+  );
+};
+Status.argTypes = {
+  ...Basic.argTypes,
+  thumbnail: { control: { disable: true } },
+  status: { control: 'radio', options: ['positive', 'warning', 'negative'] },
+  variant: { control: 'radio', options: ['default', 'folder'] },
+};
+Status.args = {
```

Comment:
I noticed that the `isNew` prop is not available to toggle. Might be good to add it to the global `args` on L34

Reply:
Updated Tiles args with different boolean controls. Disabled some of them in individual stories.